ott/tools/gaussian_mixture/gmm_demo.ipynb

```diff
@@ -1,275 +0,0 @@
-{
- "cells": [
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "UlLGoifuRk7x"
-   },
-   "source": [
-    "A quick demonstration of some Gaussian mixture related methods that we will use in estimating the $MW_2$ metric from [Delon & Desolneux](https://arxiv.org/abs/1907.05254)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "id": "KA9iIYeri_En"
-   },
-   "outputs": [],
-   "source": [
-    "import importlib\n",
-    "import typing\n",
-    "\n",
-    "import numpy as np\n",
-    "import matplotlib.pyplot as plt\n",
-    "\n",
-    "import jax\n",
-    "import jax.numpy as jnp"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {
-    "id": "LPmlrxTPaffA"
-   },
-   "outputs": [],
-   "source": [
-    "from colabtools import adhoc_import\n",
-    "\n",
-    "with adhoc_import.Google3CitcClient(\"ott-gmm\", \"geoffd\"):\n",
```

Comment:
adhoc_import brings back nice memories :)

Reply:
The reference to adhoc_import is being deleted by this pull request!